app/views/browse/build-config.html

```diff
@@ -75,8 +75,8 @@
             <div class="row" ng-if="loaded">
               <div class="col-md-12" ng-class="{ 'hide-tabs' : !buildConfig }">
                 <uib-tabset>
-                  <uib-tab active="selectedTab.summary">
-                    <uib-tab-heading>Summary</uib-tab-heading>
+                  <uib-tab active="selectedTab.builds">
```

Comment:
this will invalidate bookmarked URLs from 3.4, can you just validate if someone comes in with tab=summary that it correctly resets the url to tab=builds